components/app-core/backend/auth.go

```diff
@@ -95,6 +98,11 @@ func (p *portalProxy) loginToUAA(c echo.Context) error {
 		return err
 	}
 
+	err = p.handleSessionExpiryHeader(c)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Should lines 106-114 also get the chop?

Reply:
Yes, they should...hmmm... not sure how that happenned... good spot!

Reply:
Pushed a new commit that removes these lines.